myapp/deleteClient.js

```diff
@@ -0,0 +1,21 @@
+const request = require('request');
+const app = require('./app.js');
+
+deleteClient = function () { 
+  app.delete('/clients/:client', function (req, res) {
+    request({
+      url: 'https://api.createsend.com/api/v3.1/clients/'+req.params.client+'.json',
+      method: "DELETE",
+      },
```

Comment:
messed up indentation from line 9-17
For line 17 - remove 2 more whitespaces. This chaining is similar to the .then chaining you did in Promise.